docs/api/catalog-resource.md

```diff
@@ -257,6 +258,16 @@ The `published` timestamp is the time when the package was last listed.
 > [!Note]
 > On nuget.org, the `published` value is set to the year 1900 when the package is unlisted.
 
+#### Vulnerabilities
+
+An array of `vulnerability` objects. Each vulnerability has the following properties:
+
+Name         | Type   | Required | Notes
+------------ | ------ | -------- | -----
+advisoryUrl  | string | yes      | Location of security advisory for the package
+gitHubDatabaseKey | string | yes      | GitHub reference for security advisory
```

Comment:
Does this need to be part of the protocol? Just curious

Reply:
It would work fine without it. I'm leaving it in because we use it for the unique JSON-LD id.

Reply:
We want to avoid breaking changes in our protocol like the plague. Is that the only reason to add the GitHub database key? If so, I’d say we should remove the property (or at the very least make it not required). Otherwise, what happens if in the future we source our vulnerabilities from somewhere else other than GitHub?

Reply:
The GitHub database key is a value plumbed all the way from GitHub. IMO it's okay to be here since a) it's already in the catalog (immutable) and b) it's likely interesting for correlation to the GitHub side. The registration and catalog concerns are different. That being said, if we anticipate no one ever caring about it or using it, I agree with Loic and we can leave it undocumented and potentially remove it from catalog for leaves moving forward.

Reply:
Okay, I'll remove it--I see what you both mean (here and above). For this PR I'll remove it from the docs and I'll create a change to remove it from catalog leaves. Thanks!